examples/SoupRot.ahk

```diff
@@ -0,0 +1,11 @@
+#include ..\build\libcrypt.ahk
+
+iterationCnt:=a_tickCount
+junk:=0
+str:="Hello° µWorld"
+
+encStr:=soupRot.enc(str,iterationCnt,junk)
+decStr:=soupRot.dec(encStr,iterationCnt,junk)
+
+msgbox,,SoupRot,% encStr "`n`n" decStr
```

Comment:
Maybe show the `str` before encoding, then the encoded string and finally the decoded string.
Perhaps something like:
```ahk
msgbox,,SoupRot, Original: %str%`nEncoded: %encStr%`nDecoded: %decStr%
```

Reply:
Done!